module_utils/ha_cluster_lsr/pcs_api_v2_utils.py

```diff
@@ -214,6 +214,8 @@ def call_api_raw(
     """
     response, info = fetch_url(
         module,
+        # never use a proxy when connecting to a local unix socket (RHEL-81918)
+        use_proxy=False,
```

Comment:
ok - since this will only ever be a local unix socket (and not a TCP/IP socket), proxies should never be used.